internal/cli/alerts/settings/update.go

```diff
@@ -66,7 +66,7 @@ func UpdateBuilder() *cobra.Command {
 		Short: "Updates one alert configuration in the specified project.",
 		Args:  require.ExactArgs(1),
 		Annotations: map[string]string{
-			"args":              "alertConfigId",
+			"requiredArgs":      "alertConfigId",
 			"alertConfigIdDesc": "Unique identifier of the alert configuration you want to update.",
```

Comment:
Syntax not quite right. Need to list args, then required args, then description.
```suggestion
			"args": "alertConfigId",
			"requiredArgs":      "alertConfigId",
			"alertConfigIdDesc": "Unique identifier of the alert configuration you want to update.",
```